docs/index.rst

```diff
@@ -3,9 +3,9 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-=================================
-Welcome to oemof's documentation!
-=================================
+=======================================
+Welcome to oemof-solph's documentation!
```

Comment:
Just a note on development style: Changes like this should be made in an independent branch to be able to merge them before there is a decision about the feature branch. (And I have to confess that I am not always strict although I know better.)

Reply:
Yeah, I thought the same while making this change and then went with it anyways. I opened a separate PR for it (#952).